src/Goerli-DssSpell.t.sol

```diff
@@ -2255,23 +2043,19 @@ contract DssSpellTest is DSTest, DSMath {
 
         // Check medianizer sources
         assertEq(pip.src(), address(token));
-        assertEq(pip.orb0(), _feed0);
-        assertEq(pip.orb1(), _feed1);
+        assertEq(pip.orb0(), _medianizer1);
+        assertEq(pip.orb1(), _medianizer2);
 
         // Authorization
         assertEq(join.wards(pauseProxy), 1);
         assertEq(vat.wards(address(join)), 1);
         assertEq(clip.wards(address(end)), 1);
-        assertEq(clip.wards(address(clipMom)), _checkLiquidations ? 1 : 0);
+        if (_checkLiquidations) assertEq(clip.wards(address(clipMom)), 1);
```

Comment:
I think we should keep the previous check as it covers also the case where the wards are zero.